liquibase-core/src/main/java/liquibase/database/core/H2Database.java

```diff
@@ -174,6 +216,8 @@ public boolean isSafeToRunUpdate() throws DatabaseException {
                         || url.startsWith("jdbc:h2:mem:")
                         || url.startsWith("jdbc:h2:zip:")
                         || url.startsWith("jdbc:h2:~")
+                        || url.startsWith("jdbc:h2:./db")
+                        || url.startsWith("jdbc:h2:/")
```

Comment:
`db` should be removed, it's just a sample name of the database in the current working directory (`./`).

`jdbc:h2:C:/path/to/database` with any drive letter is also a valid local URL on Windows (similar to `jdbc:h2:/path/to/database` on other systems).

Actually the whole implementation is not reliable at all. It the intention is to exclude remote urls, the `jdbc:h2:tcp:` and `jdbc:h2:ssl:` should be excluded and all other URLs should be allowed.

Reply:
Something like
```Java
super.isSafeToRunUpdate() // <- I don't know what is this
   || (!url.startsWith("jdbc:h2:tcp:") && (!url.startsWith("jdbc:h2:ssl:");
```

Reply:
@katzyn Thanks for your input! I'll include that in the updated pull request against the 3.8.x branch.